skyportal/tests/frontend/test_favorites_page.py

```diff
@@ -26,8 +26,16 @@ def test_add_remove_favorites(driver, user, public_source):
     # little triangle you push to expand the table
     driver.click_xpath("//*[@id='expandable-button']")
 
+    driver.wait_for_xpath(
+        f"//*[contains(@data-testid, 'favorites-text-include_{public_source.id}')]",
+        timeout=10,
```

Comment:
Isn't the default timeout 10s? Wouldn't it be better to go straight to 20?

Reply:
Good idea.